test/test_aaa_run_transaction_session.py

```diff
@@ -1,41 +1,54 @@
 from concurrent.futures import ThreadPoolExecutor
-from sqlalchemy import Table, Column, MetaData, select, testing
+from sqlalchemy import Table, Column, MetaData, select, testing, text
 from sqlalchemy.ext.declarative import declarative_base
 from sqlalchemy.orm import sessionmaker
 from sqlalchemy.testing import fixtures
 from sqlalchemy.types import Integer
 import threading
 
-from cockroachdb.sqlalchemy import run_transaction
+from sqlalchemy_cockroachdb.transaction import run_transaction
+
+""" This file is named "test_aaa_run_transaction_session.py" to ensure that it is run before any
+    other tests. Testing under SQLA 1.4 revealed that it ran by itself just fine, but if *any*
+    other test ran before it (even the corresponding "core" test) then it would crash with
```

Comment:
This feels slightly janky to me. Do we know why this is the case? Is there a way we can mitigate the  issue without relying on the order in which tests are run?

Reply:
> This feels slightly janky to me. 

Agreed.

> Do we know why this is the case? Is there a way we can mitigate the issue without relying on the order in which tests are run?

I hope we can do that eventually. In my experience, issues like this sometimes get resolved after a while when others encounter similar behaviour and a pattern starts to emerge. What we know at the moment is:

1. The test isn't broken *per se* in 1.4 because it passes when it is run first.
2. It worked okay under its previous name with 1.3 so this has something to do with 1.4, particularly how this specific test interacts with ORM (session, etc.) and the test environment (plugins, fixtures, and whatnot).

Perhaps for now we can flag it with a `# TODO: Why does this happen?` and keep an eye out for similar issues. Ideally if we could come up with an MCVE to reproduce it using one of the internal dialects (SQLite, PostgreSQL, et al.) it would get a lot more traction as a SQLAlchemy issue.

Reply:
@arulajmani - Please let me know if you're okay with this revision, at least for now. Reconciling with current master will likely be a PITA due to the (significant) reorganization of files, so it would be good to get it done before another PR comes along and makes a bigger mess to clean up later. :(

Reply:
Okay, I think it's worth capturing this information in a comment and adding a TODO there to keep an eye out.

Reply:
Sounds good.